### ISSUE TITLE
Doesn't load or find posts

### ISSUE THREAD

**Issue by aharris88** [krasnoukhov/octodmin#10]
When I load the page, it doesn't load any posts. It just has a spinner and there's an error in the JavaScript console. I'm probably just missing a configuration, but I'm not sure what.

![screen shot 2015-03-27 at 1 55 22 pm](https://cloud.githubusercontent.com/assets/3266363/6875980/f63641c6-d488-11e4-9b96-2915d0f16b3d.png)

**Comment by krasnoukhov** [comment]
Damn, this looks like the date format you use in your posts isn't easily parseable. Could I get a sample?

**Comment by aharris88** [comment]
Yeah, so I don't normally put the date in the front matter of my posts. If I do, this is the format:

`date: 2012-08-23`

Otherwise I just name it according to Jekyll's format and leave it out of the front matter. So, for example, I have a post named `2012-09-19-wordpress-plugin-to-get-free-coffee.markdown` and if I put `{{ page.date }}` in the post, then it prints this on the page: `2012-09-19 00:00:00 -0600`

**Comment by krasnoukhov** [comment]
I've just checked and both empty date and one like "2012-08-23" works. This should be something else, I think

**Comment by krasnoukhov** [comment]
I've released 0.3.5, please try that. I believe it resolves your problem

**Comment by agarrharr** [comment]
That works great! btw, I've added octodmin to my [list of services for static sites](https://github.com/aharris88/services-for-static-websites#online-editors-for-github-pages)

**Comment by krasnoukhov** [comment]
Thanks!
